src/systemdhook.c

```diff
@@ -158,7 +158,7 @@ static int chperm(const char *id, const char *path, const char *label, int uid,
 				return -1;
 			}
 			if (label != NULL && (strcmp("", label))) {
-				if (setfilecon (full_path, label) < 0) {
+				if ((is_selinux_enabled() > 0) && (setfilecon (full_path, label) < 0)) {
```

Comment:
I just wonder if it makes sense to instead do something like:
```
if ((is_selinux_enabled() <=0)
    pr_perror("Selinux is not enabled and needs to be to securely configure the container.")
````
or some such.  I'm just worried about silently failing if selinux isn't enabled.  

Regardless, if @mrunalp is hip, then I'm hip with this.

Reply:
I could see adding a message about < 0, although I don't believe I have ever seen this. 
== 0 means that the admin of the system booted the machine with SELinux disabled, so we don't do any SELinux actions.  I don't see a reason to blog about it, it is not an error but the way the system is configured.